### PR TITLE
[#17] Update the mtime of parent collections (4-2-stable)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -171,7 +171,25 @@ namespace
             // Elevate privileges so that all users can create hard links.
             ix::scoped_privileged_client spc{conn};
 
-            return rsPhyPathReg(&conn, &input);
+            const auto ec = rsPhyPathReg(&conn, &input);
+
+            // Update the parent collections mtime.
```
Comment (trel) [inline src/main.cpp:176]: collection's

Comment (korydraughn) [reply src/main.cpp]: done

**Comment by korydraughn** [comment]
added #

**Comment by korydraughn** [comment]
hold on merging this

**Comment by korydraughn** [comment]
added #

**Pull request merged**